daemon/cmd/ipam_test.go

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+//go:build !privileged_tests && integration_tests
```

Comment:
```suggestion
//go:build !privileged_tests
```

This is not an integration test relying on e.g. the kvstore, so it doesn't need the tag.